src/ripple/shamap/impl/SHAMap.cpp

```diff
@@ -682,7 +682,7 @@ bool SHAMap::delItem (uint256 const& id)
             if (bc == 0)
             {
                 // no children below this branch
-                prevHash = uint256 ();
+                prevHash = SHAMapHash{};
```

Comment:
Possible micro-optimization... Is it cheaper to call `prevHash.zero()`?  Or will the compiler optimize away the construction of the temporary `SHAMapHash`?